src/applications/vaos/tests/appointment-list/components/CanceledAppointmentsList.unit.spec.js

```diff
@@ -300,33 +245,25 @@ describe.skip('VAOS Component: CanceledAppointmentsList', () => {
   });
 
   it('should show video appointment on gfe text', async () => {
-    const startDate = moment.utc();
-    const data = {
-      id: '1234',
-      currentStatus: 'CANCELLED BY CLINIC',
-      status: APPOINTMENT_STATUS.cancelled,
-      kind: 'telehealth',
-      start: startDate.format(),
-      locationId: '983GC',
-      telehealth: { vvsKind: VIDEO_TYPES.gfe },
-    };
-    const appointment = createMockAppointmentByVersion({
-      version: 0,
-      ...data,
-    });
-
-    mockAppointmentInfo({ va: [appointment] });
+    // Arrange
+    const startDate = moment();
+    const appointment = mockBaseCanceledAppointment(startDate);
+    appointment.attributes.kind = 'telehealth';
+    appointment.attributes.currentStatus = 'CANCELLED BY CLINIC';
+    appointment.attributes.telehealth = { vvsKind: VIDEO_TYPES.adhoc };
+    appointment.attributes.extension = { patientHasMobileGfe: true };
```

Comment:
I updated how GFE appointments are specified by following the logic [here](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/vaos/appointment-list/components/AppointmentsPage/AppointmentCard.jsx#L24-L25). This is different from the existing code but I'm assuming this mechanism changed at some point and is expected.